exercises/bowling/tests/bowling.rs

```diff
@@ -47,20 +51,22 @@ fn a_game_score_is_err_if_fewer_than_ten_frames_have_been_rolled() {
 }
 
 #[test]
+#[ignore]
 fn a_game_score_is_err_if_more_than_ten_frames_have_been_rolled() {
```

Comment:
I thought about that, but decided it was probably unnecessary. But I could be convinced otherwise.